src/lib/email.ts

```diff
@@ -0,0 +1,33 @@
+import { env } from 'process'
+
+import fetch from 'node-fetch'
+
+interface EmailRequestOptions<T> {
+  from: string
+  to: string
+  subject: string
+  body: T
+}
+
+export const sendEmail = async <T>(template: string, { from, to, subject, body }: EmailRequestOptions<T>) => {
+  if (typeof window !== 'undefined') {
+    throw new TypeError('Netlify Emails is not available in the browser, please use a Netlify Function')
+  }
+
+  if (!env.NETLIFY_EMAILS_SECRET) {
+    throw new Error('It does not look like Netlify Emails is enabled for this site.')
+  }
+
+  await fetch(`${env.URL}/.netlify/functions/emails/${template}`, {
```

Comment:
considering this is a fetch call to another function, maybe there's a way to make this a method call instead of a network request? might be hard to do bundling-wise, cc @eduardoboucas

Reply:
This would be brilliant, would love to do this if possible.

Ultimately could do some dynamic imports I imagine to achieve this?

cc @lemusthelroy